RoaringBitmap/src/test/java/org/roaringbitmap/longlong/TestRoaring64Bitmap.java

```diff
@@ -802,6 +803,28 @@ public void testAnd() {
     assertEquals(123, left.select(0));
   }
 
+  @Test
+  public void testAndDisjoint() {
+    // There are no shared values between these maps.
+    final long[] leftData = new long[]{1076595327100L, 1074755534972L, 5060192403580L, 5060308664444L};
+    final long[] rightData = new long[]{3470563844L};
+
+    Roaring64Bitmap left = Roaring64Bitmap.bitmapOf(leftData);
+    Roaring64Bitmap right = Roaring64Bitmap.bitmapOf(rightData);
+
+    left.and(right);
+
+    Roaring64Bitmap swapLeft = Roaring64Bitmap.bitmapOf(rightData);
```

Comment:
testing the reverse to ensure that the operation is commutative since it wasn't before.